tests/test_delete.py

```diff
@@ -34,6 +29,13 @@ def test_delete_bucket_success(deletable_bucket):
     assert delete_result.unwrap() is None
 
 
+def test_delete_bucket_non_modal(deletable_bucket):
+    """Test that we get exception raised when we try to create existing."""
+    assert delete_bucket(  # type: ignore
```

Comment:
Shouldn't this method return `True`? Therefore, you call `delete_bucket(...)` and get the confirmation that the bucket was actually deleted.

Reply:
its a good question, we used to have some of these return True, but the google library returns None, and we do too since as long as there are no exception raised its 100% deleted. Also because it will never return False its not really a boolean return value.